shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -348,6 +387,9 @@ public Options getOptions() {
     optEndRowExclusive.setArgName("end-exclusive");
     scanOptRow = new Option("r", "row", true, "row to scan");
     scanOptColumns = new Option("c", "columns", true, "comma-separated columns");
+    scanOptCfOptions = new Option("cf", "column-family", true, "Column Family");
+    scanOptColQualifier = new Option("cq", "column-qualifier", true, "Column Qualifier");
```

Comment:
```suggestion
    scanOptCfOptions = new Option("cf", "column-family", true, "column family");
    scanOptColQualifier = new Option("cq", "column-qualifier", true, "column qualifier");
```
To match the rest of the options. Could also add more detail such as `column family to scan`.